custom_components/flashforge_adventurer_3/protocol.py

```diff
@@ -1,30 +1,38 @@
 import asyncio
 from codecs import StreamReader, StreamWriter
 import logging
-import os
 import re
 from typing import Optional, Tuple, TypedDict
 
 logger = logging.getLogger(__name__)
 
 BUFFER_SIZE = 1024
 TIMEOUT_SECONDS = 5
-STATUS_COMMAND = '~M601 S1'
-PRINT_JOB_INFO_COMMAND = '~M27'
-TEMPERATURE_COMMAND = '~M105'
+STATUS_COMMAND = '~M119\r\n'
```

Comment:
Looks like the status command is no longer used. I guess we can remove it? (I was never sure if it was needed in the first place 😛)